<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Smilei</class>
 <widget class="QMainWindow" name="Smilei">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmileiQt</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
    <item>
     <layout class="QGridLayout" name="plotGrid">
      <item row="0" column="0">
       <widget class="QWidget" name="plot" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="allBack">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="back">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="playStop">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="forward">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="allForward">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>316</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionDir"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDiagnostics">
    <property name="title">
     <string>Diagnostics</string>
    </property>
    <addaction name="actionFirst"/>
    <addaction name="actionSecond"/>
    <addaction name="actionThird"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDiagnostics"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDir">
   <property name="text">
    <string>&amp;Choose directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFirst">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>first</string>
   </property>
  </action>
  <action name="actionSecond">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>second</string>
   </property>
  </action>
  <action name="actionThird">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>third</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>&lt;</string>
   </property>
   <property name="toolTip">
    <string>back</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="text">
    <string>&gt;</string>
   </property>
   <property name="toolTip">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionFirst_2">
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
   <property name="toolTip">
    <string>go to first timestep</string>
   </property>
  </action>
  <action name="actionLast">
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
   <property name="toolTip">
    <string>Go to last timestep</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
